### PR TITLE
Updated history sample for mv3

### PR DESCRIPTION
**Pull request by IanStanion-google**
Addresses https://github.com/GoogleChromeLabs/Extension-Docs/issues/85

### CODE REVIEW

File: api-samples/history/showHistory/manifest.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Typed URL History",
+  "version": "1.3",
+  "description": "Reads your history, and shows the top ten pages you go to by typing the URL.",
```
Comment (jpmedley) [inline api-samples/history/showHistory/manifest.json:4]:
```suggestion
  "description": "Reads your history and shows the top ten pages you go to by typing the URL.",
```

Comment (jpmedley) [reply api-samples/history/showHistory/manifest.json:4]: You generally don't use a comma if there are only two in the list.

File: api-samples/history/showHistory/showHistory.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# chrome.history
+
+This sample demonstrates using the `chrome.history` API to display the user's most visited pages in the extension popup.
```
Comment (jpmedley) [inline api-samples/history/showHistory/showHistory.md:3]:
```suggestion
This sample demonstrates using the [`chrome.history`](https://developer.chrome.com/docs/extensions/reference/history/) API to display the user's most visited pages in the extension popup.
```

Comment (jpmedley) [reply api-samples/history/showHistory/showHistory.md:3]: We should probably be linking to the API we're demonstrating.

File: api-samples/history/showHistory/showHistory.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+## Overview
+
+This extension makes use of the `chrome.history.search` API to scrape the browser's history and count occurances of each visited URL.
```
Comment (jpmedley) [inline api-samples/history/showHistory/showHistory.md:7]:
```suggestion
This extension calls `chrome.history.search()` to scrape the browser's history and count occurrences of each visited URL.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: api-samples/history/showHistory/typedUrls.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Recently Typed URLs</title>
+    <style>
+      body {
+        min-width: 250px;
+      }
+    </style>
+    <script src="typedUrls.js"></script>
```
Comment (AmySteam) [inline api-samples/history/showHistory/typedUrls.html:10]: Can you rename the script file to use lowercase and dashes? Same with the HTML file. Thanks!

Also, can you move the script tag into the body?

[File name style guide
](https://developers.google.com/style/filenames#:~:text=Guidelines%20for%20names,-Make%20file%20and&text=Use%20hyphens%2C%20not%20underscores%2C%20to,page%20names%20such%20as%20document1.
)

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Pull request merged**